algorithms/qaoa/knapsack/qaoa_knapsack.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    with ExecutionSession(
</pre><p>Maybe it would make sense to reuse the same execution session. To span multiple blocks you can create a global instance and then explicitly close it, instead of using <code>with ExecutionSession</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>